docs/install/2-install-server-with-podman-and-systemd.rst

```diff
@@ -0,0 +1,328 @@
+.. _install-server-with-podman:
+
+Install Inmanta with Podman and Systemd
+***************************************
+
+This page explains how to setup an Inmanta OSS orchestration server using Podman and Systemd.
+This guide assumes you already have `Podman <http://podman.io/>`_ installed on your machine and that you are running a Linux distribution with Systemd.
+
+.. note::
+    The full setup should be doable without any root privilege (rootless) on the host, running the orchestrator with an unprivileged user.  
+
+.. warning::
+    The setup described below assumes you already have a PostgreSQL instance available that the orchestrator can use for its persistent storage.  If it is not the case, 
+    please :ref:`jump to the end of this document<install-postgresql-with-podman>`, where we explain to you how to easily deploy a database using Postman and Systemd.
+
+
+Pull the image
+##############
+
+.. only:: oss
+
+    Use ``podman pull`` to get the desired image:
+
+    .. code-block:: sh
+
+        podman pull ghcr.io/inmanta/orchestrator:latest
+
+
+    This command will pull the latest version of the Inmanta OSS Orchestrator image.
+
+.. only:: iso
+
+    Step 1: Log in to container registry
+    -------------------------------------
+
+    Connect to the container registry using your entitlement token.
+
+    .. code-block:: console
+
+        $ podman login containers.inmanta.com
+        Username: containers
+        Password: <your-entitlement-token>
+
+        Login Succeeded
+        $
+
+
+    Replace ``<your-entitlement-token>`` with the entitlement token provided with your license.
+
+
+    Step 2: Pull the image
+    ----------------------
+
+    Use ``podman pull`` to get the desired image:
+
+    .. code-block:: sh
+        :substitutions:
+
+        podman pull containers.inmanta.com/containers/service-orchestrator:|version_major|
+
+
+    This command will pull the latest version of the Inmanta Service Orchestrator image.
```

Comment:
For the ISO docs build, the tag is a major version number. So this statement is only true within the major version this documentation build is for.

Reply:
Is it better now?